.gitignore

```diff
@@ -1,6 +1,8 @@
 bin/configlet
 bin/configlet.exe
 
+bin/Generator
```

Comment:
These changes should likely go into another pr, just saying this for future cases.

Reply:
Yeah, no problem, I can do it now. Here we are:

https://github.com/exercism/swift/pull/892